### PR TITLE
[13.0][MIG] connector_elasticsearch

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Comment by rousseldenis** [comment]
@lmignon @simahawk

**Comment by rousseldenis** [comment]
@OCA/search-engine-maintainers

File: setup/connector_elasticsearch/setup.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import setuptools
+
+setuptools.setup(
+    setup_requires=['setuptools-odoo'],
+    odoo_addon=True,
```
Comment (lmignon) [inline setup/connector_elasticsearch/setup.py]: you must pin the version of elasticsearch
https://github.com/OCA/search-engine/blob/12.0/setup/connector_elasticsearch/setup.py#L6

**Review by lmignon — CHANGES_REQUESTED** [review]

File: setup/connector_elasticsearch/setup.py (hunk 1 above)
Comment (rousseldenis) [reply setup/connector_elasticsearch/setup.py]: Done

**Pull request closed without merging**